activerecord/lib/active_record/connection_adapters/mysql/schema_definitions.rb

```diff
@@ -54,6 +54,42 @@ def unsigned_float(*args, **options)
         def unsigned_decimal(*args, **options)
           args.each { |name| column(name, :unsigned_decimal, options) }
         end
+
+        def geometry(*args, **options)
+          return multi_geometry(*args, **options) if options[:multi] == true
+          args.each { |name| column(name, :geometry, options) }
+        end
+
+        def point(*args, **options)
+          return multi_point(*args, **options) if options[:multi] == true
+          args.each { |name| column(name, :point, options) }
+        end
+
+        def linestring(*args, **options)
+          return multi_linestring(*args, **options) if options[:multi] == true
+          args.each { |name| column(name, :linestring, options) }
+        end
+
+        def polygon(*args, **options)
+          return multi_polygon(*args, **options) if options[:multi] == true
+          args.each { |name| column(name, :polygon, options) }
+        end
+
+        def multi_geometry(*args, **options)
+          args.each { |name| column(name, :multi_geometry, options) }
+        end
+
+        def multi_point(*args, **options)
+          args.each { |name| column(name, :multi_point, options) }
+        end
+
+        def multi_linestring(*args, **options)
+          args.each { |name| column(name, :multi_linestring, options) }
+        end
+
+        def multi_polygon(*args, **options)
+          args.each { |name| column(name, :multi_polygon, options) }
+        end
```

Comment:
Don't think it makes sense exposing those multi columns both via an option and a method, I'd go with the option across the board:

```ruby
def point(*args, multi: false, **options)
  args.each { |name| column(name, multi ? :multi_point : :point, options) }
end
```

Then the same for the other versions.